src/courtroom.cpp

```diff
@@ -3762,15 +3762,30 @@ void Courtroom::set_scene(QString f_desk_mod, QString f_side)
   }
 }
 
-void Courtroom::set_self_offset(QString p_list) {
+void Courtroom::set_self_offset(QString p_list, QString p_effect) {
     QStringList self_offsets = p_list.split("&");
+    QStringList play_effect = p_effect.split("|");
     int self_offset = self_offsets[0].toInt();
     int self_offset_v;
     if (self_offsets.length() <= 1)
       self_offset_v = 0;
     else
       self_offset_v = self_offsets[1].toInt();
     ui_vp_player_char->move(ui_viewport->width() * self_offset / 100, ui_viewport->height() * self_offset_v / 100);
+
+    int effect_x;
+    int effect_y;
+
+    if (ao_app->get_effect_property(play_effect[0], current_char, "ignore_offset") == "true") {
+      effect_x = (ui_viewport->width() * self_offset / 100) + ui_viewport->x();
+      effect_y = (ui_viewport->height() * self_offset_v / 100) + ui_viewport->y();
+      ui_vp_effect->move(effect_x, effect_y);
```

Comment:
Wait this doesn't ignore offsets this is just the same code as below

Reply:
That's what I get for finishing code at 4AM, copying it somewhere else and not checking what I copied. Lemme fix that.

Reply:
Working on AO at 4 am?
Unfortunately, I can relate 🙂